manage.py

```diff
@@ -21,6 +21,12 @@
         print("Error accessing Zulip secrets; manage.py in production must be run as the zulip user.")
         sys.exit(1)
 
+    # Performance Hack: We make the pika.adapters.twisted_connection
+    # module unavailable, to save ~100ms of import time for most Zulip
+    # management commands for code we don't use.  The correct
+    # long-term fix for this will be to get a setting integrated
+    # upstream to disable pika importing this.
+    sys.modules['pika.adapters.twisted_connection'] = {}
```

Comment:
This interferes with Django’s autoreload because `dict` is not hashable. Using a legitimate empty module instead fixes it: `sys.modules['pika.adapters.twisted_connection'] = types.ModuleType('pika.adapters.twisted_connection')`.